components/tikv_util/src/sys/mod.rs

```diff
@@ -1,5 +1,45 @@
 // Copyright 2017 TiKV Project Authors. Licensed under Apache-2.0.
 
+mod cgroup;
+use self::cgroup::CGroupSys;
+
+const KB: u64 = 1024;
```

Comment:
Why not use `tikv_util::config::KB` directly?

Reply:
done